kuka_rsi_driver/config/joint_trajectory_controller_config_6_axis_kl.yaml

```
@@ -0,0 +1,18 @@
+/**/joint_trajectory_controller:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)